packages/fxa-auth-server/lib/payments/paypal/helper.ts

```diff
@@ -677,4 +677,55 @@ export class PayPalHelper {
 
     return;
   }
+
+  /**
+   * Attempts to refund all of the invoices passed, provided they're created via PayPal
+   * This will invisibly do nothing if the invoice is not billed through PayPal, so be mindful
+   * if using it elsewhere and need confirmation of a refund.
+   */
+  public async refundInvoice(invoice: Stripe.Invoice) {
+    this.log.debug('PayPalHelper.refundInvoice', {
+      invoiceId: invoice.id,
+    });
+    const minCreated = Math.floor(
+      new Date().setDate(new Date().getDate() - MAX_REFUND_DAYS) / 1000
+    );
+    if (invoice.collection_method !== 'send_invoice') {
+      throw new Error('Invoice is not a Paypal invoice');
```

Comment:
micro nit: not sure if this is user facing, but&hellip; PayPal (to match thrown error on L710)? 🤷 
```suggestion
      throw new Error('Invoice is not a PayPal invoice');
```

but pretty small nit since its already pretty dispersed through the codebase (see PayPalHelper, vs PaypalTransactionId).
```sh
git grep -Phio "paypal" -- "*.ts" | sort -i | uniq -c
 132 PAYPAL
 409 PayPal
 426 Paypal
   6 payPal
 547 paypal
```

Reply:
Yeahhh, we need to standardize this everywhere. It's a problem within libs too.